src/lib/reasoners/adt_rel.ml

```diff
@@ -86,38 +106,43 @@ module Domain = struct
     | Ty.Tadt (name, params) ->
       (* Return the list of all the constructors of the type of [r]. *)
       let cases = Ty.type_body name params in
+      let hash = get_hash ty in
```

Comment:
Does not look like `get_hash` is only used by the legacy frontend after all?

Reply:
I meant that the purpose of this wrapper is to preserve the support for the legacy frontend but we have to call it to retrieve the hash function computed in `Nest` too.